src/schemas/vuorovaikutus.ts

```diff
@@ -6,6 +6,32 @@ import filter from "lodash/filter";
 
 const validTimeRegexp = /^([0-1]?[0-9]|2[0-4]):([0-5]?[0-9])$/;
 
+export const palauteSchema = Yup.object().shape({
+  etunimi: Yup.string().notRequired().max(100),
+  sukunimi: Yup.string().notRequired().max(100),
+  sahkoposti: Yup.string().notRequired().email(),
```

Comment:
Pitäis varmaan laittaa epäkelvolliseen emailiin varoitusteksti. Ja sit pitäisi varoitustekstit laittaa kääntymään ruotsiksi...